databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -171,6 +186,199 @@ def update_share_cli(api_client, name, new_name, comment, owner,
+              help="Updates the shared table to shared data object represented in JSON file.")
+@click.option('--json', default=None, type=JsonClickType(),
+              help="Updates the shared table to shared data object represented in JSON.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def add_share_table_cli(api_client, share, table, shared_as, comment,
+                     partitions, cdf, no_cdf, json_file, json):
+    """
+    Adds a shared table.
+
+    The public specification for the JSON request is in development.
+    """
+    cdf_enabled = None
+    if cdf is not None and no_cdf is not None:
+        raise ValueError("You can only pass in either --cdf or --no-cdf and not both.")
+    
+    if cdf is not None:
+        cdf_enabled = cdf
+    elif no_cdf is not None:
+        cdf_enabled = no_cdf
+
+    if (shared_as is not None or comment is not None or partitions is not None or
+        cdf is not None or no_cdf is not None):
+        if (json_file is not None) or (json is not None):
+            raise ValueError('Cannot specify JSON if any other flags are specified')
+        data = { 
+            'updates': [
+                {
+                    'action': 'ADD',
+                    'data_object': shared_data_object(
+                        name=table,
+                        shared_as=shared_as,
+                        comment=comment,
+                        cdf_enabled=cdf_enabled,
+                        partitions=json_loads(partitions),
+                    )
+                }
+            ]
+        }
+        share_json = UnityCatalogApi(api_client).update_share(share, data)
+        click.echo(mc_pretty_format(share_json))
+    else:
+        json_cli_base(json_file, json, lambda d: UnityCatalogApi(api_client).update_share(share, { 
+            'updates': [
+                {
+                    'action': 'ADD',
+                    'data_object': d,
+                }
+            ]
+        }))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update a shared table.')
+@click.option('--share', required=True,
+              help='Name of the share to update.')
+@click.option('--table', required=True,
+              help='Full name of the table to update from share.')
+@click.option('--shared-as', default=None,
+              help='New name of the table inside the share.')
+@click.option('--comment', default=None,
+              help='New comment of the table inside the share.')
+@click.option('--partitions', default=None, type=JsonClickType(),
+              help='New partition specification of the table inside the share represented in JSON.')
+@click.option('--cdf', is_flag=True, default=None,
+              help='Enables change data feed of the table inside the share.')
+@click.option('--no-cdf', is_flag=True, default=None,
+              help='Enables change data feed of the table inside the share.')
```

Comment:
Maybe these options can be shared, like in https://github.com/databricks/databricks-cli/blob/main/databricks_cli/unity_catalog/cred_cli.py#L70